build/build.ps1

```diff
@@ -73,64 +73,6 @@ function GetVersion([string] $name) {
   throw "Failed to find $name in Versions.props"
 }
 
-function GetVsWhereExe{
-  $vswhereVersion = GetVersion("VSWhereVersion")
-  $vsWhereDir = Join-Path $ToolsRoot "vswhere\$vswhereVersion"
-  $vsWhereExe = Join-Path $vsWhereDir "vswhere.exe"
-
-  if (!(Test-Path $vsWhereExe)) {
-    Create-Directory $vsWhereDir
-    Write-Host "Downloading vswhere"
-    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
-    Invoke-WebRequest "https://github.com/Microsoft/vswhere/releases/download/$vswhereVersion/vswhere.exe" -OutFile $vswhereExe
-  }
-  
-  return $vsWhereExe
-}
-
-function InstallVSIX([string] $vsixExpInstalleExe, [string] $rootsuffix, [string] $vsInstallDir, [string] $pathToVSIX){
-  $rootedPath = [System.IO.Path]::GetFullPath($vsInstallDir)
-  & $vsixExpInstalleExe /rootSuffix:$rootsuffix /vsInstallDir:"$rootedPath" $pathToVSIX
-}
-
-function LocateVisualStudio {
-  if ($InVSEnvironment -and $log) {
-    Write-Host "Using Visual Studio from VSINSTALLDIR environment variable: $env:VSINSTALLDIR"
-    return $env:VSINSTALLDIR
-  }
-
-  $vsWhereExe = GetVsWhereExe
-  $vsInstallDir = & $vsWhereExe -all -latest -prerelease -property installationPath    -requires Microsoft.Component.MSBuild -requires Microsoft.VisualStudio.Component.VSSDK -requires Microsoft.Net.Component.4.6.TargetingPack -requires Microsoft.VisualStudio.Component.Roslyn.Compiler
-  $vsVersion    = & $vsWhereExe -all -latest -prerelease -property installationVersion -requires Microsoft.Component.MSBuild -requires Microsoft.VisualStudio.Component.VSSDK -requires Microsoft.Net.Component.4.6.TargetingPack -requires Microsoft.VisualStudio.Component.Roslyn.Compiler
-
-  if (!(Test-Path $vsInstallDir)) {
-    throw "Failed to locate Visual Studio (exit code '$lastExitCode')."
-  }
-
-  if ($log) {
-    Write-Host "Using VS version $vsVersion in directory: $vsInstallDir"
```

Comment:
It'd still be useful to output this somewhere when `/diagnostic` is specified.

Reply:
I had intentions of doing this via the command prompt logo itself which logs version. I'll do some follow up here around this.